courses/machine_learning/deepdive2/recommendation_systems/solutions/content_based_by_hand.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<p>This isn't really an official lab, more of a fun thing Mike made so students can get a little intuition, otherwise if we want to make it a full lab then it will need learning objectives.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/691/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Ah whoops, I didn't see Michael's profile in the blame layer, but I see it now. Sorry Mike!

Chris mentioned possible teaching it next week, so offered to dust it off. Up to the team if they want to make it official or not. I'll add some LOs just in case.

Reply:
<p>Cool, this is the solution version right (since the code for the TODOs is still there)? If so, do you plan on cloning this and deleting the TODO code before Chris' ASL or would you like me to do it? Doesn't matter to me :)</p>